web/vibe/web/common.d

```diff
@@ -482,14 +482,18 @@ package struct PathAttribute
 /// private
 package struct NoRouteAttribute {}
 
-/// Private struct describing the origin of a parameter (Query, Header, Body).
-package struct WebParamAttribute {
+/**
+ * This struct contains a mapping between the name used by HTTP (field)
+ * and the parameter (identifier) name of the function.
+ */
+public struct WebParamAttribute {
 	import vibe.web.internal.rest.common : ParameterKind;
 
+	/// The type of the WebParamAttribute
 	ParameterKind origin;
-	/// Parameter name
+	/// Parameter name (Server side)
```

Comment:
What about "function parameter name" to make this even more clear?

Reply:
will do